src/formatISODuration/index.ts

```diff
@@ -26,8 +27,21 @@ import requiredArgs from '../_lib/requiredArgs/index'
  *   seconds: 0
  * })
  * //=> 'P39Y2M20DT0H0M0S'
+ *
+ * @example
+ * // Format the given duration as ISO 8601 string with trailing zeros removed
+ * const result = formatISODuration({
+ *   years: 39,
+ *   months: 0,
+ *   days: 0,
+ *   hours: 7
+ * });
+ * //=> 'P39Y7H'
```

Comment:
Just found this leftover here:

```suggestion
 * //=> 'P39YT7H'
```

Reply:
Good catch!